### PR TITLE
Arsenal - Add grenade fuse time stat

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- title, https://github.com/acemod/ACE3/issues/3594#issuecomment-625617544

### CODE REVIEW

File: addons/arsenal/stringtable.xml [hunk 1]
```diff
@@ -1189,5 +1189,11 @@
             <Chinese>有效時間</Chinese>
             <Czech>Time to live</Czech>
         </Key>
+        <Key ID="STR_ACE_Arsenal_StatExplosionTime">
+            <English>Fuse Time</English>
+        </Key>
+        <Key ID="STR_ACE_Arsenal_DetonatesOnImpact">
+            <English>Detonates on impact</English>
```
Comment (bux) [inline addons/arsenal/stringtable.xml:1196]:
```suggestion
            <English>Detonates on impact</English>
            <German>Aufschlagzünder</German>
```

File: addons/arsenal/stringtable.xml [hunk 2]
```diff
@@ -1189,5 +1189,12 @@
             <Chinese>有效時間</Chinese>
             <Czech>Time to live</Czech>
         </Key>
+        <Key ID="STR_ACE_Arsenal_StatExplosionTime">
+            <English>Fuse Time</English>
```
Comment (Elgin675) [inline addons/arsenal/stringtable.xml:1193]:
```suggestion
            <English>Fuse Time</English>
            <French>Retard avant détonation</French>
```

File: addons/arsenal/stringtable.xml [hunk 3]
```diff
@@ -1189,5 +1189,12 @@
...
+        </Key>
+        <Key ID="STR_ACE_Arsenal_DetonatesOnImpact">
+            <English>Detonates on impact</English>
+            <German>Aufschlagzünder</German>
```
Comment (Elgin675) [inline addons/arsenal/stringtable.xml:1197]:
```suggestion
            <German>Aufschlagzünder</German>
            <French>Détonation à l'impact</French>
```

**Comment by Elgin675** [comment]
I think there's a problem with the `Fuse time` stats on some grenades.
For example, the vanillas infrared grenades :
- `I_IR_Grenade`
- `O_IR_Grenade`
- `I_E_IR_Grenade`
- `B_IR_Grenade`
- `O_R_IR_Grenade`
indicates 310s before fuse.

**Comment by Fernir2000** [comment]
@Elgin675 I think this is because these are destroyed after 310s, they emit light before that.

File: addons/arsenal/functions/fnc_statTextStatement_explosionTime.sqf [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Example:
+ * [_stats, _config] call ace_arsenal_fnc_statTextStatement_explosionTime
+ *
+ * Public: No
+ */
+
+params ["", "_config"];
+
+private _explosionTime = getNumber (configFile >> "CfgAmmo" >> getText (_config >> "ammo") >> "explosionTime");
```
Comment (PabstMirror) [inline addons/arsenal/functions/fnc_statTextStatement_explosionTime.sqf:21]: Handle IR grenades
```
    class IRStrobeBase: GrenadeCore {
        timeToLive = 300;
        explosionTime = 310;
```


```suggestion
private _ammo = getText (_config >> "ammo");
private _timeToLive = getNumber (configFile >> "CfgAmmo" >> _ammo >> "timeToLive");
private _explosionTime = getNumber (configFile >> "CfgAmmo" >> _ammo >> "explosionTime");

if (_explosionTime > _timeToLive) exitWith { "-" };
```

**Pull request merged**